lib/telegram-bot.js

```diff
@@ -3,6 +3,7 @@ const Telegraf = require("telegraf");
 const { config } = require("./config");
 const { BehaviorSubject } = require("rxjs");
 const { map, distinctUntilChanged } = require("rxjs/operators");
+const moment = require("moment-timezone");
```

Comment:
You used moment-timezone, but then used a fixed timezone: `Europe/Rome`.
I think it should be fine to just use moment, so that the local time will be based on the server timezone. We may need to add instructions on how to set this for Docker.

Reply:
Yeah, it should be taken from the environment but I don't know how to do it in Node